WebsiteGreetz/src/test/java/WebsiteTest.java

```diff
@@ -24,56 +22,53 @@ private void logIn(WebDriver driver) throws InterruptedException {
         Thread.sleep(5000);
     }
 
-    private boolean findFavorite(WebDriver driver, int countOfFavorites) {
-        if (driver.findElements(By.className("center full-size")).size() > countOfFavorites)
-            return true;
+    private boolean findFavorite(int countOfFavorites) throws InterruptedException {
+        Thread.sleep(5000);
+        return driver.findElements(By.className("full-size")).size() > countOfFavorites;
+    }
 
-        return false;
+    private boolean find5and90euro() throws InterruptedException{
```

Comment:
this method doesn't throw any Exception

Reply:
Yes, there was thread.sleep() at first, then I got it works without it as well and delete thread.sleep(). But forgot "throws...."

Reply:
It is.I will replace the method with variable.

I will change the name.